src/serverlist.cpp

```diff
@@ -25,6 +25,71 @@
 #include "serverlist.h"
 
 /* Implementation *************************************************************/
+
+// --- CServerListEntry ---
+CServerListEntry CServerListEntry::parse ( QString strHAddr,
+                                           QString strLHAddr,
+                                           QString sName,
+                                           QString sCity,
+                                           QString strCountry,
+                                           QString strNumClients,
+                                           bool    isPermanent )
+{
+    bool ok;
```

Comment:
Could you please describe in a comment what this bool does?

Reply:
It's used by `String.toInt(&ok)` - checking for zero isn't good enough validation in some places.